### ISSUE TITLE
Do not intern strings literals (constants)

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#569]
Interning string constants (literals) does not make any sense as they are already interned.

So we should report something like:
```c#
var x = string.Intern("some text");
```

And propose to fix it as following:
```c#
var x = "some text";
```
